okta/resource_okta_app_bookmark.go

```diff
@@ -40,6 +40,7 @@ other arguments that changed will be applied.`,
 			"authentication_policy": {
 				Type:        schema.TypeString,
 				Optional:    true,
+				Computed:    true,
```

Comment:
No. It will allow the provider to set the authentication_policy to default if it is not specified. I want to deprecate authentication_policy, but it will have to be major version change